tests/test_options.py

```diff
@@ -462,14 +462,15 @@ def cli_alt(warnings):
     result = runner.invoke(cli_alt, ['-w'])
     assert result.output == 'True\n'
 
+
 @pytest.mark.parametrize('option_args,expected', [
     (['--aggressive', '--all', '-a'], 'aggressive'),
-    (['--first', '--second', '--third', '-a', '-b', '-c'], 'first'),
-    (['--apple', '--banana', '--cantaloupe', '-a', '-b', '-c'], 'apple'),
+    (['--first', '--second', '--third', '-a', '-b', '-c'], 'second'),
```

Comment:
This makes me so dizzy to read. 😂 

I understand that we're going back to documented behavior, but this is pretty much the perfect example of why I think [this comment of yours is spot on](https://github.com/pallets/click/issues/1140#issuecomment-432291544). We should really just tell people to write down what they mean...